integrations.rst

```diff
@@ -8,35 +8,13 @@ This topical list of build systems, IDEs, and CI platforms provides information
 packages can be consumed, created, and continuously deployed/tested with each, as applicable.
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 3
```

Comment:
Better to keep it to two levels, otherwise the index will be too long and will contain too detailed entries.
```suggestion
   :maxdepth: 2
```